app/controllers/BusinessMatchingController.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import connectors.BusinessMatchingConnector
+import javax.inject.Inject
+import models.IndividualMatchingSubmission
+import play.api.libs.json.{JsResult, JsValue, Json}
+import play.api.mvc.{Action, ControllerComponents, Result}
+import uk.gov.hmrc.domain.Nino
+import uk.gov.hmrc.http.HttpResponse
+import uk.gov.hmrc.play.bootstrap.controller.BackendController
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class BusinessMatchingController @Inject()(
+                                            businessMatchingConnector: BusinessMatchingConnector,
+                                            override val controllerComponents: ControllerComponents)
+                                          (implicit executionContext: ExecutionContext) extends
+  BackendController(controllerComponents) {
+
+  def individualMatchingSubmission(nino: Nino): Action[JsValue] = Action(parse.json).async {
+    implicit request =>
+      val individualMatchingSubmissionResult: JsResult[IndividualMatchingSubmission] =
+        request.body.validate[IndividualMatchingSubmission]
+
+      individualMatchingSubmissionResult.fold(
+        invalid = _ => Future.successful(BadRequest("")),
```

Comment:
Provide a message about what caused BadRequest?

Reply:
I'm not sure we want to be passing up the validation results. The caller will check their submission.